contracts/BatchExchangeViewer.sol

```diff
@@ -348,6 +349,19 @@ contract BatchExchangeViewer {
         return updateSellTokenBalance(element, sellTokenBalance);
     }
 
+    function getTokenInfo(uint16 tokenId)
+        public
+        view
+        returns (
+            address,
+            string memory symbol,
+            uint8 decimals
+        )
+    {
+        ERC20Detailed token = ERC20Detailed(batchExchange.tokenIdToAddressMap(tokenId));
+        return (address(token), token.symbol(), token.decimals());
```

Comment:
Would it be worth returning the `tokenID` from the  `BatchExchange` here as well?

Reply:
That would be redundant as it's already part of the method invocation.